modules/ROOT/pages/sap/sap-connector-studio.adoc

```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../../assets/images/
 
-Install the SAP Connector and configure the connector in Anypoint Studio. 
+Install SAP Connector and configure the connector in Anypoint Studio. 
```

Comment:
You don't install connectors in Mule 4, you add them to a project. Please fix this point in the headings. All Studio 7 connectors are provided in the sense that you don't need to restart Studio to access them -- just set a dependency in the POM file using the internal search feature or using Exchange.